opencti-platform/opencti-front/lang/front/en.json

```
@@ -2673,5 +2673,10 @@
   "Operational": "Operational",
   "Strategic": "Strategic",
   "Link created": "Link created",
-  "Public dashboard": "Public dashboard"
+  "Public dashboard": "Public dashboard",
+  "Multiple Values Entered Edit with the TT button": "Multiple Values Entered Edit with the TT button",
```

```suggestion
  "Multiple Values Entered Edit with the TT button": "Multiple values entered. Edit with the TT button",
```
A point or comma is missing to have a full understanding as end user. Also the platform does not have much text starting with upper case on each words, so I removed some of them for consistency. (Feel free to change the key too if you want, but then you will need to change in all translation files).

---

@aHenryJard - I can clean it up, based on your suggestion.

---

@aHenryJard - Adapted to your requested change. I'm noticing that the en.json is missing a key/translation line from a PR last week that was merged. The key/translation missing is "Unless specific groups are selected, user will be created with default groups only."  I can add it into the PR here - if you like - to correct it? (Technically, since it's in the en.json file, it will just work, but I believe the practice is to have all potentially translated strings in all files).